RecoLuminosity/LumiProducer/test/analysis/plugins/BuildFile.xml

```diff
@@ -0,0 +1,47 @@
+<library   name="PCCNTuplerPlugin" file="*.cc">
```

Comment:
I think this file is not needed.
the code is already compiled by  RecoLuminosity/LumiProducer/test/BuildFile.xml